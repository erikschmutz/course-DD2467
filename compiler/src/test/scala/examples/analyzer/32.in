type color = RED | GREEN | RGB of int * int * int;;


(*====================
type color : color (RED | GREEN | RGB of int * int * int)
- :()
====================*)