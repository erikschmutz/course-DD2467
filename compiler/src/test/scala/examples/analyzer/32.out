AnalyzerResult(Unit(),Enviroment(List(TypeEntry(Identifier(color),Constraints(color,List(Constraint(RED,None,List()),Constraint(GREEN,None,List()),Constraint(RGB,Some(Tuple(List(Integer(),Integer(),Integer()),List())),List())),List())))))